src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -214,10 +214,32 @@ private void testEditSessionAction() throws Exception {
         feedbackEditPage.clickEditSessionButton();
         editedSession.setInstructions(new Text("Made some changes"));
         feedbackEditPage.editFeedbackSession(editedSession.getEndTime(), editedSession.getStartTime(),
-                                        editedSession.getInstructions(), editedSession.getGracePeriod());
+                                        editedSession.getInstructions(), editedSession.getGracePeriod(), true);
 
         String expectedString = "The end time for this feedback session cannot be earlier than the start time.";
         feedbackEditPage.verifyStatus(expectedString);
+
+        ______TS("test discard changes to feedback session");
+
+        // Make changes to Feedback session
```

Comment:
There are some unexpected differences between the expected and actual HTML when I `verifyHtmlPart` against `/instructorFeedbackEditDiscardChangesOk.html`. You can see the diffs here. https://www.diffchecker.com/8OeggAEz

These seem like they are caused by small differences like the behaviour of `enableEditFS()` in JS as opposed to simply reloading the page.

Requesting advice on best path forward. I was thinking, in this case, if it would be okay to accept a third file to simplify this?